api/openapi-spec/v3/apis__discovery.k8s.io__v1_openapi.json

```diff
@@ -140,7 +140,7 @@
         "description": "EndpointPort represents a Port used by an EndpointSlice",
         "properties": {
           "appProtocol": {
-            "description": "appProtocol represents the application protocol for this port. This field follows standard Kubernetes label syntax. Un-prefixed names are reserved for IANA standard service names (as per RFC-6335 and https://www.iana.org/assignments/service-names). Non-standard protocols should use prefixed names such as mycompany.com/my-custom-protocol.",
+            "description": "Used as a hint for implementations to configure the protocol used between the implementation and the application it exposes. This field follows standard Kubernetes label syntax. Valid values are either:\n\n* Un-prefixed protocol names - reserved for IANA standard service names (as per RFC-6335 and https://www.iana.org/assignments/service-names).\n\n* Kubernetes standard names:\n  * 'k8s.io/http2' - http2 over cleartext, aka 'h2c'. https://www.rfc-editor.org/rfc/rfc7540\n  * 'k8s.io/grpc' - grpc traffic - see https://github.com/grpc/grpc/blob/v1.51.1/doc/PROTOCOL-HTTP2.md\n  * 'k8s.io/tcp' - plain tcp traffic\n\n* Other protocols should use prefixed names such as mycompany.com/my-custom-protocol.",
```

Comment:
is introducing `\n\n` characters , should it?

Reply:
I do have extra space lines between different bullets. so yes